README.md

```diff
@@ -19,46 +19,78 @@ In some cases, it is possible to prefetch the static resources of an application
 The prefetch builder generates a `ngxPrefetch.js` file that should be included in the HTML page of the entry point. When run, it dynamically [creates `<link>` tags](https://developer.mozilla.org/en-US/docs/Web/HTTP/Link_prefetching_FAQ) for each static resource (such as JS and CSS files) so that the browser can prefetch them during idle times. This will improve the Page Load Time of the application when it is actually loaded.
```

Comment:
We could give a bit more explanation here: "This will improve the Page Load Time of the application when it is actually loaded" => This will improve the Page Load Time of the application when the application it accessed for the first time. Then the browser caching will take over, until a new version of the application is deployed or the browser application cache is cleared.